db/migrate/20160621204706_rename_stil_id_on_user.rb

```diff
@@ -0,0 +1,5 @@
+class RenameStilIdOnUser < ActiveRecord::Migration
+  def change
+    rename_column(:users, :stil_id, :student_id)
```

Comment:
I think we should remove stil_id instead of renaming. That will make the transition faster, since people will have to update to LUCAT (if they want to rent the car, candidate, etc). Also, this is the perfect time to do it.